### ISSUE TITLE
Training Betas and Gammas of the batch normalization

### ISSUE THREAD

**Issue by andrearama** [jonbruner/generative-adversarial-networks#4]
Am I wrong, or in your current implementation you are not updating the betas and the gammas of the batch normalization layers (during of course the training) ?

Bests,
Andrea

**Comment by adeshpande3** [comment]
Thanks for bringing this up. Looking at [documentation](https://www.tensorflow.org/api_docs/python/tf/contrib/layers/batch_norm), it seems that the is_training parameter is set to true by default, so the updates should be happening (Although because we're explicitly training d_vars and g_vars in our optimizer declerations, I do have a suspicion that possibly the updates are not occurring).

**Comment by andrearama** [comment]
Thanks for the reply Adit.
Your doubt is in fact my main point: displaying the Beta value (gamma is actually not used here, since the parameter  scale is by default put as False) it can be seen that it always keeps equal to zero. 
A simple modification to overcome this problem would be to rename the variables 'bn1' and 'bn2' to 'g_bn1' and 'g_bn2'.

**Comment by jonbruner** [comment]
Thanks for pointing this out, Andrea, and thanks for confirming, Adit. I've changed the scope parameters in the generator's batch_norm layers, and it appears that they're updating during training now. Code is pushed; let me know if you encounter any problems with the changes.
